workshop/exercises/README.md

```diff
@@ -3,13 +3,13 @@
 ## Docker compose environment
 
 ### Start
-To start the docker composition, from ./workshop/docker folder, use the command:
+To start the Docker composition, from `workshop/exercises`, run the following command:
 
 ```bash
 docker-compose up -d
 ```
 
-This will start two docker containers, one with our pygeoapi (reachable from the browser at [this address](http://localhost:5000/)) and one with Elasticsearch (localhost:9200), which can be used as storage backend.
+This will start two Docker containers, one with our pygeoapi (reachable from the browser at [this address](http://localhost:5000/)) and one with Elasticsearch (localhost:9200), which can be used as storage backend.
```

Comment:
Think all ES text should be moved to exercises/elastic. That dir needs cleanup as well (everything that was under docker/ES is obsolete/not required).